src/pages/clients/create/Create.tsx

```diff
@@ -59,8 +59,8 @@ export function Create() {
   });
 
   useEffect(() => {
-    if (blankClient?.data.data) {
-      setClient(blankClient.data.data);
+    if (blankClient) {
+      setClient(blankClient.data?.data);
     }
```

Comment:
When I go to the client creation page, the whole application crashes. The reason for this is reading data from an undefined value. I recommend that you redo this part in the same way as for the rest of the project using `useBlankClientQuery`. Let me know your thoughts.

Reply:
This page doesn't use /client/create query?